app/src/main/java/fr/free/nrw/commons/wikidata/WikiBaseClient.kt

```diff
@@ -81,19 +81,20 @@ class WikiBaseClient
         fun addLabelsToWikidata(
             fileEntityId: Long,
             languageCode: String?,
-            captionValue: String?,
-        ): Observable<MwPostResponse> =
-            csrfToken().switchMap { editToken ->
+            captionValue: String?
+        ): Observable<MwPostResponse> {
+            return csrfToken().switchMap { editToken ->
                 wikiBaseInterface.addLabelstoWikidata(
                     PAGE_ID_PREFIX + fileEntityId,
                     editToken,
                     languageCode,
-                    captionValue,
+                    captionValue
                 )
             }
+        }
 
         private fun csrfToken(): Observable<String> =
-            Observable.fromCallable {
-                csrfTokenClient.getTokenBlocking()
-            }
+                Observable.fromCallable {
+                    csrfTokenClient.getTokenBlocking()
+                }
```

Comment:
Please perform such stylistic changes in a separate style-only pull request. Thanks!